### PR TITLE
Add topas output files

### PR DESCRIPTION
**Pull request by nbassler**

### CODE REVIEW

File: notebooks/xy-plot.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from topas2numpy import BinnedResult
+
+ntracks = BinnedResult('/home/villads/2022_DCPT_LET/data/topas/results/output/plan_2/Scoring_protonD_XYBox.csv')
```
Comment (grzanka) [inline notebooks/xy-plot.py:5]: avoid user paths

**Comment by grzanka** [comment]
@Villadslj please fix conflicts

File: notebooks/xy-plot.py (hunk 1 above)
Comment (Villadslj) [reply notebooks/xy-plot.py]: I have changed it now

**Comment by grzanka** [comment]
@Villadslj can you fix the conflicts ?
Also - the notebooks directory should contain Jupyter notebooks, not the python source code files.

**Comment by grzanka** [comment]
Python scripts are ok, but should go into `src` directory

**Comment by Villadslj** [comment]
Should be fixed now @grzanka

**Comment by Villadslj** [comment]
So it is ready for merge @grzanka ?

**Comment by grzanka** [comment]
> So it is ready for merge @grzanka ?

I will check in the afternoon.

**Review by grzanka — APPROVED** [review]

**Pull request merged**